### ISSUE TITLE
Gmail message history

### ISSUE THREAD

**Issue by airainchoc** [edotassi/AmazMod#114]
Shen I receive an email, it shows well un thé notification :
- icon
- app name
- message title
- message body
- canned answers

But in the notification history, it doesn't show message body nor canned answers

**Comment by lfom** [comment]
Dup of #103?

**Comment by airainchoc** [comment]
Oops, sorry

**Comment by airainchoc** [comment]
Not really, as I think it could be great to add ability to read message / reply from history (we can't always reply when the first notification comes)

Idk if it is feasible, but it sure could be useful

**Comment by lfom** [comment]
Then you want to have a separate notifications history on watch, where you can reply later from, right? Then it's not only for Gmail. If you can, please also fix the text, I think it's mixed up with French...

**Comment by airainchoc** [comment]
Well I think it could be two enhancements:
- be able to read full Gmail message in History
- be able to reply from history (I don't know if it's even possible to add this function)

(Sorry for french / english mix, auto correct on phone...)

**Comment by lfom** [comment]
First one is a dup of #103 more or less, and the other one is already planed, but with no date to release...
